src/model/consent/consent.ts

```diff
@@ -45,9 +45,18 @@ export interface Consent {
 export class ConsentDB {
   // Knex instance
   private Db: Knex
+  // Nullable fields which don't allow for explicit null upsert
+  private nullProtectedProps: string[]
 
   public constructor (dbInstance: Knex) {
     this.Db = dbInstance
+    this.nullProtectedProps = [
```

Comment:
`nullProtectedProps` should be a static property

Reply:
Modified in a recent commit.